### ISSUE TITLE
Not support signed manifest in filterEvents

### ISSUE THREAD

**Issue by ScorpioCPH** [vmware/harbor#901]
If you are reporting a problem, please make sure the following information are provided:
1)Version of docker engine and docker-compose
```
$ docker --version
Docker version 1.12.1, build 6f9534c
```
```
$ docker-compose --version
docker-compose version 1.8.0, build f3628c7
```
2)Config files of harbor, you can get them by packaging "Deploy/config" directory
```
N/A
```
3)Log files, you can get them by package the /var/log/harbor/ 
```
N/A
```

After pushing a new repository to Harbor, no push event triggered. By debugging i got a media type `application/vnd.docker.distribution.manifest.v1+prettyjws` 

This is a signed manifest but is missing in `filterEvents` function.

Reference: [Image Manifest](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-1.md)

**Comment by ScorpioCPH** [comment]
@reasonerjt, Hi, I have fixed this issue, which branch should i submit to now, dev or master?

**Comment by reasonerjt** [comment]
neither, we're doing a heavy refactory, early nextweek you can submit your PR to dev branch...

**Comment by reasonerjt** [comment]
@ScorpioCPH, you can submit your PR to dev branch now.

**Comment by ScorpioCPH** [comment]
OK, thanks.

**Comment by reasonerjt** [comment]
fixed in #923
